package.json

```
@@ -1,33 +1,52 @@
 {
   "name": "rds-contracts",
```

We have to release it under @rsksamrt org:
```suggestion
  "name": ""@rsksmart/rif-marketplace-storage-pinning",
```